packages/react/src/ActionList/Item.tsx

```diff
@@ -137,6 +137,12 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       },
     }
 
+    const listItemStyles = {
+      display: 'flex',
+      // show between 2 items
+      ':not(:first-of-type)': {'--divider-color': theme?.colors.actionListItem.inlineDivider},
```

Comment:
I believe this was added to fix some styles when `<button>` was added. @broccolinisoup might have more context here!

Reply:
I don't remember sorry 😢 It has been a while since I pushed that draft PR. We can comment it out and see how the stlyes are different to figure out if it still works fine in this PR.

Reply:
I went ahead and removed it, and it seems like it was needed for cases where [a divider is used](https://primer-657d0c9a88-13348165.drafts.github.io/storybook/?path=/story/components-actionlist-features--item-dividers) and the "button" semantics are applied. I think it's worth keeping for this 🤔